### PR TITLE
Fix #3610: Documentation for Hint Editor

### PR DESCRIPTION
**Pull request by kevintab95**
I think the issue was submitted in the main oppia project rather than oppia.github.io . This PR documents the editor functionality for hints.

### CODE REVIEW

File: pages/TheExplorationEditor.html [hunk 1]
```diff
@@ -104,4 +104,27 @@
...
+  </h4>
+
+  <p>
+    Once you've specified what Oppia should do for answers, you can specify
+    hints for the current card. You can add a list of hints that the learner can
+    make use of when he/she is stuck. To prevent learners from abusing this
+    feature, a hint can be requested only if one of the following conditions are
+    satisfied:
+    <ol>
+      <li>The learner may request a hint if he/she got an answer wrong at-least
```
Comment (seanlip) [inline pages/TheExplorationEditor.html]: The learner submitted a wrong answer at least once.

File: pages/TheExplorationEditor.html [hunk 2]
```diff
@@ -104,4 +104,27 @@
...
+        once.</li>
+      <li>Or it has been more than 30 seconds since the learner has submitted an
```
Comment (seanlip) [inline pages/TheExplorationEditor.html]: It has been more than 30 seconds since the learner last submitted an answer.

**Review by seanlip — CHANGES_REQUESTED** [review]

**Comment by seanlip** [comment]
@kevintab95 -- could you finish this please? Also it would be great if at least one of @tjiang11 @joshuacano @kevinlee12 could review this.

File: pages/TheExplorationEditor.html [hunk 3]
```diff
@@ -104,4 +104,27 @@
...
+    <a id="Hints" class="anchor" href="#Hints" aria-hidden="true">
+      <span class="octicon octicon-link"></span>
+    </a>
+    Hints
+  </h4>
+
+  <p>
+    Once you've specified what Oppia should do for answers, you can specify
+    hints for the current card. You can add a list of hints that the learner can
+    make use of when he/she is stuck. To prevent learners from abusing this
```
Comment (tjiang11) [inline pages/TheExplorationEditor.html]: You can add a list of hints that the learner can use when stuck.

**Review by tjiang11 — CHANGES_REQUESTED** [review]

**Review by tjiang11 — APPROVED** [review]

**Review by seanlip — APPROVED** [review]

**Pull request merged**